### PR TITLE
update target attribute

### PR DESCRIPTION
**Pull request by morganen**

### CODE REVIEW

File: content/md/assets/assets-transformation.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Say we have a `packshots` asset family. In its structure, it has 2 media file at
 
 The PIM can automatically generate the thumbnail version of your main image for you, and this, thanks to the definition of a transformation!
 
-![Assets Transformations](../img/Assets_Transformations.png)
+![Assets Transformations](../img/Assets_TransformationsV2.png)
```
Comment (bgouraud) [inline content/md/assets/assets-transformation.md:21]: We use the same image twice on the same page?

File: content/md/assets/assets-transformation.md [hunk 2]
```diff
@@ -57,12 +57,22 @@ It will be used in error messages whenever your transformation failed to generat
 
 ## Source file
 
-The `source` property allows you to define in which attribute value is stored the media file you want to use as the source file for your transformation. For instance: `main_image`.
+The `source` property allows you to define in which attribute value is stored the **media file** you want to use as the source file for your transformation. For instance: `main_image`.
```
Comment (camillebourjade) [inline content/md/assets/assets-transformation.md:60]:
```suggestion
The `source` property allows you to define in which attribute value the **media file** you want to use as the source file for your transformation is stored. For instance: `main_image`.
```

File: content/md/assets/assets-transformation.md [hunk 3]
```diff
@@ -57,12 +57,22 @@ It will be used in error messages whenever your transformation failed to generat
...
+
+::: warning
+Only the `media file` attribute types can be used as sources. The transformation feature is not available for `media link` attribute types.
```
Comment (camillebourjade) [inline content/md/assets/assets-transformation.md:63]:
```suggestion
The `media file` attribute types are the only ones that can be used as sources. The transformation feature is not available for `media link` attribute types.
```

File: content/md/assets/assets-transformation.md [hunk 4]
```diff
@@ -57,12 +57,22 @@ It will be used in error messages whenever your transformation failed to generat
...
+:::
 
 ## Target file
 
 The `target` property allows you to define in which attribute value the PIM should generate the new variation. For instance: `Thumbnail_variation`.
 
+::: info
+As soon as an attribute is defined as a target attribute of a transformation, it becomes disabled (read only). It means that you cannot modify or remove it afterwards. If the target attribute changes, then the attribute previously defined as target becomes enabled and you are able to modify or remove it.
```
Comment (camillebourjade) [inline content/md/assets/assets-transformation.md:71]:
```suggestion
As soon as an attribute is defined as a target attribute of a transformation, it becomes disabled (read-only). It means that you can no longer modify or remove it. If the target attribute changes, then the attribute previously defined as the target becomes enabled and you can modify or remove it.
```

File: content/md/assets/assets-transformation.md (hunk 1 above)
Comment (morganen) [reply content/md/assets/assets-transformation.md:21]: yes, it's normal. But if it's weird, I can remove it.

**Pull request merged**